### DISCUSSION TITLE
Build Bailed

### DISCUSSION THREAD

**Discussion by wangtaooo** [AvaotaSBC/.github#6] [AvaotaOS]
"AvaotaOS/build_dir//bootloader.bin) failed: No such file or directory"

**Comment by wangtaooo** [comment]
编译指令 "sudo bash build_all.sh     -b avaota-a1     -m https://mirrors.ustc.edu.cn/debian     -v bookworm     -t cli     -u avaota     -p avaota     -s avaota     -k no     -i no     -o no     -e no"

**Comment by chainsx** [comment]
Please provide the operating system of the compilation host you encountered the issue with, the host architecture, and the complete log.
